src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -1044,14 +958,6 @@ private List<String> getEquipmentsIds(UUID networkUuid, String variantId, List<S
         }
     }
 
-    public LineMapData getLine(UUID networkUuid, String variantId, String lineId) {
-        Line line = getNetwork(networkUuid, PreloadingStrategy.NONE, variantId).getLine(lineId);
-        if (line == null) {
-            throw new ResponseStatusException(HttpStatus.NOT_FOUND);
-        }
-        return toMapData(line, true);
-    }
-
     public List<GeneratorMapData> getGenerators(UUID networkUuid, String variantId, List<String> substationsId) {
```

Comment:
see remarks on NetworkMapController : some methods can be removed

Reply:
Done in an another US